### DISCUSSION TITLE
Synthetic Data Package

### DISCUSSION THREAD

**Discussion by hollandjg** [AutoResearch/autora#414] [Show and Tell]
Hi all, there’s a new pre-release of the synthetic data package with yesterday’s feedback (v1.0.0a6).

Code: https://github.com/AutoResearch/autora-synthetic-data
Package: https://pypi.org/project/autora-synthetic-data 
Docs: https://autoresearch.github.io/autora-synthetic-data/example/

Does this structure make more sense? 
This might be especially interesting for: @younesStrittmatter @musslick @blinodelka

**Comment by younesStrittmatter** [comment]
This is really cool

One tiny detail: The folder and import path are called "synthetic", while the repo and package are called "synthetic-data". We might want to make that consistent and choose one of both.

**Reply by hollandjg** [reply]
autora-synthetic?

**Reply by musslick** [reply]
Yes, let's go with "**autora-synthetic**" for now with the package name because technically, it's more than data.

**Reply by hollandjg** [reply]
Done

**Comment by musslick** [comment]
This looks great! From a first glance, it might be worth adding our developer documentation for guiding contributions?

**Reply by hollandjg** [reply]
How about this: 
https://github.com/AutoResearch/autora-synthetic-data/blob/main/CONTRIBUTING.md

**Reply by younesStrittmatter** [reply]
I like the documentation but am not 100% sure about the placement. If the CONTRIBUTING.md file is in the root (instead of docs) of the repository, we can not pull it into the autora-parent documentation. I'll test Sebastian's suggestion for a mkdocs plugin (#415). Then we could easily include the CONTRIBUTION.md in a page of the docs (also, we could make the README.md the indexes, which would be neat), but based on experience, I wonder if it'll work with the multirepo.

**Reply by hollandjg** [reply]
do we need the contributing guidelines in the online docs themselves? won't people who want to contribute be looking at the repo anyway?

**Reply by musslick** [reply]
@hollandjg 
I think it would be neat to have all of the main contributor documentation inside the main doc, as it is the case with many other packages. 
So we would have a subitem in the doc for:

Contributor Guidelines
- Overview
- Contributions to autora
- Contributions to autora-core
- Contributions to autora-synthetic

At least for autora-core and autora-synthetic, we can provide some basic info but them point them (via a link) to the documentation at the respective repository.

**Reply by hollandjg** [reply]
TODOS: Update contribution guidelines for:
- [x] autora + overview
- [x] autora-core
- [x] autora-synthetic

**Comment by hollandjg** [comment]
We seem to be done here – I'll close this discussion.
